english/matrix/matrix.md

```diff
@@ -25,3 +25,5 @@ While you don't believe that you might be the One,
 
 [You crawl through the hole you just made and...] (matrix2/matrix2.md)
 
+[There is no darkness - or fluffiness - without the Light.] (../matrix2inception/matrix2inception.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.